### PR TITLE
synchronized refactor: move l1_parallel to a folder

### PR DESCRIPTION
**Pull request by joanestebanr**
### What does this PR do?
Refactor to group L1 parallel stuff in a folder

### Reviewers

Main reviewers:
- @ARR552

### CODE REVIEW

File: synchronizer/l1_parallel_sync/l1_workers.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func (w *workers) String() string {
 	return result
 }
 
-func newWorkers(ethermans []EthermanInterface, cfg workersConfig) *workers {
+func newWorkers(ethermans []L1EthermanInterface, cfg workersConfig) *workers {
```
Comment (ARR552) [inline synchronizer/l1_parallel_sync/l1_workers.go]: Why this change?? Etherman only reads info from L1

File: synchronizer/l1_parallel_sync/l1_worker_etherman_test.go [hunk 2]
```diff
@@ -25,10 +25,9 @@ func TestExploratoryWorker(t *testing.T) {
 
 	l1Config := etherman.L1Config{
 		L1ChainID:                 1337,
-		ZkEVMAddr:                 common.HexToAddress("0x8dAF17A20c9DBA35f005b6324F493785D239719d"),
-		RollupManagerAddr:         common.HexToAddress("0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e"),
+		ZkEVMAddr:                 common.HexToAddress("0x610178dA211FEF7D417bC0e6FeD39F05609AD788"),
 		PolAddr:                   common.HexToAddress("0x5FbDB2315678afecb367f032d93F642f64180aa3"),
-		GlobalExitRootManagerAddr: common.HexToAddress("0x8A791620dd6260079BF849Dc5567aDC3F2FdC318"),
+		GlobalExitRootManagerAddr: common.HexToAddress("0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6"),
```
Comment (ARR552) [inline synchronizer/l1_parallel_sync/l1_worker_etherman_test.go]: please, undo this change. Addresses were correct

**Review by ARR552 — CHANGES_REQUESTED** [review]

File: synchronizer/l1_parallel_sync/l1_workers.go (hunk 1 above)
Comment (joanestebanr) [reply synchronizer/l1_parallel_sync/l1_workers.go]: the interface is the specific for L1Parallel, you are right that is a bad name. I'll rename to something more specific

File: synchronizer/l1_parallel_sync/l1_worker_etherman_test.go [hunk 3]
```diff
@@ -27,7 +27,7 @@ func TestExploratoryWorker(t *testing.T) {
 		L1ChainID:                 1337,
 		ZkEVMAddr:                 common.HexToAddress("0x610178dA211FEF7D417bC0e6FeD39F05609AD788"),
```
Comment (ARR552) [inline synchronizer/l1_parallel_sync/l1_worker_etherman_test.go]: Still some addresses removed. You should keep all of them

**Review by ARR552 — CHANGES_REQUESTED** [review]

File: synchronizer/l1_parallel_sync/l1_worker_etherman_test.go (hunk 3 above)
Comment (joanestebanr) [reply synchronizer/l1_parallel_sync/l1_worker_etherman_test.go]: restored all address

**Review by ARR552 — APPROVED** [review]
